docker-compose.yaml

```diff
@@ -1,5 +1,10 @@
----
-version: '3.0'
+version: '3.8'
+
+volumes:
+  chunks-data:
+  contiguous-data:
+  headers-data:
+  db-data:
```

Comment:
I like this structure, but we need to continue to use volume mounts specified by environment variables by default for these.

Reply:
I need to look deeper into combining those two 👍